packages/nimble-components/src/table/specs/table-column-specs/table-column-mapping.md

```diff
@@ -92,66 +94,69 @@ Cons:
 
 ### API
 
-#### Icon column element:
-
-_Component Name_
-
--   `nimble-table-column-icon`
-
-_Props/Attrs_
-
--   `field-name`: string
--   `key-type`: 'string' | 'number' | 'boolean'
-
-_Content_
-
--   column title (icon)
--   1 or more `nimble-mapping-icon` or `nimble-mapping-spinner` elements
-
-#### General mapping column element:
+#### Enum column element:
 
 _Component Name_
 
--   `nimble-table-column-enum-text`
+-   `nimble-table-column-enum`
 
 _Props/Attrs_
 
 -   `field-name`: string
 -   `key-type`: 'string' | 'number' | 'boolean'
--   `fractional-width`: number (defaults to 1)
--   `min-pixel-width`: number (defaults to minimum supported by table)
+-   `fixed-width-mode`: enum - `flexibleWidth` (default) | `iconWidth` - When set to `iconWidth`, the column will have a fixed width that makes the column the appropriate width to render only a single icon in the cell. This should only be set when the header contains a single icon (no text) and none of the child mapping elements will result in text being rendered in a cell.
```

Comment:
Chatted with molly to clarify a couple things.

- The column should have the fractional width mixin for the fractional-width and min-pixel-width properties. They were unintentionally omitted.
- Rounded on keeping the enum but tweaking the names to avoid repeating "width", to use the term icon-size which aligns with the design token name, and to not give an explicit name of "flexible" to the default mode but just call it default (i.e. the standard sizing behavior from all other columns).

So concretely:
`width-mode: default | icon-size`